### DISCUSSION TITLE
How can I Customization tooltip ?

### DISCUSSION THREAD

**Discussion by gerry-momo** [imaNNeo/fl_chart#1023] [Q&A]
This is my ToolTip
![image](https://user-images.githubusercontent.com/73962678/169633820-33ab2283-128d-419f-8fb2-126fb9032707.png)
This is what I need
![image](https://user-images.githubusercontent.com/73962678/169633835-7cb45bbf-b5f0-4ded-b812-7c6493a0a021.png)

I found that I could not achieve the desired effect with LineTooltipItem, showing the time at the top and then the item name and value.
Because I have multiple parameters

**Comment by imaNNeo** [comment]
Please follow this issue: #72
